clients/gtest/trsm_gtest.cpp

```diff
@@ -140,7 +140,7 @@ Arguments setup_trsm_arguments(trsm_tuple tup)
     arg.transA_option = side_uplo_transA_diag[2];
     arg.diag_option   = side_uplo_transA_diag[3];
 
-    arg.timing = 1;
+    arg.timing = 0;
```

Comment:
Not clear how this gets replicated, you want timing on for the bench?

Reply:
This is turning timing off for hipblas-test, I'm not sure why it was set to 1 here to begin with, especially since it didn't do anything previously.